daphne/src/messages/mod.rs

```diff
@@ -971,6 +987,43 @@ impl Decode for HpkeCiphertext {
     }
 }
 
+/// A plaintext input share.
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
```

Comment:
Hmm, do we need `Serialize, Deserialize` for this struct?

Reply:
we don't, I'll remove it